sample/src/main/AndroidManifest.xml

```diff
@@ -8,9 +8,12 @@
 
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
     <uses-permission android:name="android.permission.INTERNET"/>
-    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
 
+    <!-- not needed as of api 23 -->
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
```

Comment:
deleted previous: But will update to do what docs say:

```
<uses-permission
        android:name="android.permission.WRITE_EXTERNAL_STORAGE"
        android:maxSdkVersion="22" />
```

It looks like according to https://developers.google.com/maps/documentation/android-api/config a future version will not need it at all.